api/camunda/camunda-openapi.yaml

```
@@ -1157,6 +1278,48 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+
+  /deployments:
+    post:
+      tags:
+        - Resources
+      summary: Deploy resources
+      description: |
+        Deploys one or more resources (e.g. processes, decision models, or forms). This is an atomic call, i.e. either all resources are deployed or none of them are.
```

```suggestion
        Deploys one or more resources (e.g. processes, decision models, or forms). 
        This is an atomic call, i.e. either all resources are deployed or none of them are.
```

category: endpoint description sentence

owner: me

This one isn't a _required_ change....but I think a line break between these two sentences fits, and the second sentence is not necessary for the meta description.

---

https://github.com/camunda/camunda/pull/22101